.github/workflows/lint_test_workflow.yml

```
@@ -0,0 +1,36 @@
+on:
```

Please add human readable names so they come up in the github actions and checks areas more readable.

```suggestion
name: Code Quality
on:
```